src/redisraft.c

```diff
@@ -555,6 +555,66 @@ static void handleAsking(RedisRaftCtx *rr, RedisModuleCtx *ctx)
     RedisModule_ReplyWithSimpleString(ctx, "OK");
 }
 
+static void handleClientUnblock(RedisRaftCtx *rr, RedisModuleCtx *ctx, RaftRedisCommand *cmd)
+{
+    raft_session_t id;
+    bool error = false;
+
+    if (cmd->argc != 3 && cmd->argc != 4) {
+        RedisModule_ReplyWithError(ctx, "ERR syntax error");
```

Comment:
Can we use `RM_WrongArity()` here.

Reply:
its not what clientCommand returns.  It actually calls addReplySubcommandSyntaxError() which is what I was hinting to myself there, need to fix it up to be

"unknown subcommand or wrong number of arguments for '%.128s'. Try %s HELP"

where I should make "ERR unknown subcommand or wrong number of arguments for 'unblock'.  Try client HELP"